pkg/crc/machine/host.go

```diff
@@ -0,0 +1,74 @@
+package machine
+
+import (
+	"fmt"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/machine/bundle"
+	"github.com/code-ready/crc/pkg/libmachine"
+	libmachinehost "github.com/code-ready/crc/pkg/libmachine/host"
+	"github.com/pkg/errors"
+)
+
+type virtualMachine struct {
+	name string
+	*libmachinehost.Host
```

Comment:
Why this doesn't have any name association something like `host *libmachinehost.Host` ?

Reply:
This allows to use the methods from `libmachinehost.Host` on a `virtualMachine` instance, even if I don't remember if I make use of this.
```
  vm := virtualMachine{}
  vm.Stop()
  vm.Kill()
```
(the last 2 methods are from `pkg/libmachine/host/`)

Reply:
So if we put `host *libmachinehost.Host` then we need to do `vm.host.Stop()` which is kind of cryptic.